### PR TITLE
Add Base64 to glossary

### PR DESCRIPTION
**Pull request by apinnick**
Add Base64 to the glossary.

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@apinnick Please add the context in the PR description.

@redhat-documentation/ccs-style-council PTAL at this PR and provide feedback/approval.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc [hunk 1]
```diff
@@ -86,6 +86,17 @@
 
 *See also*: xref:distinguished-name[distinguished name]
 
+[[base64]]
+==== image:images/yes.png[yes] Base64 (noun, adjective)
+*Description*: _Base64_ is a group of binary-to-text encoding schemes that transforms binary data into a sequence of printable characters, limited to a set of 64 unique characters.
```
Comment (rolfedh) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc:91]:
```suggestion
*Description*: _Base64_ is a group of binary-to-text encoding schemes that converts binary data into a sequence of 64 unique, printable characters.
```

**Review by rolfedh — APPROVED** [review]
LGTM - I've also offered an optional suggestion to make the definition more concise.

**Comment by apinnick** [comment]
@rolfedh  Can someone merge this?

**Comment by rolfedh** [comment]
Hi @apinnick, I'm glad to help. I've posted [a request for review](https://redhat-internal.slack.com/archives/C04HKR61MN1/p1717444361243409) on the #forum-ccs-style.

**Comment by apinnick** [comment]
> Hi @apinnick, I'm glad to help. I've posted [a request for review](https://redhat-internal.slack.com/archives/C04HKR61MN1/p1717444361243409) on the #forum-ccs-style Slack channel.

@rolfedh  Thanks. I read the [Contributing guide](https://github.com/redhat-documentation/supplementary-style-guide/blob/main/CONTRIBUTING.md) but there is no guidance about what to do if no one reviews or approves a PR.

**Review by dfitzmau — APPROVED** [review]

**Comment by rolfedh** [comment]
I unintentionally merged this before receiving the third approval. We discussed this in today's council meeting and nobody raised any objections. Nevertheless, if anyone has follow up comments, please make them here and I'll address them in a follow-up PR.

**Comment by bergerhoffer** [comment]
Not a problem - giving the third approval verbally here now!

**Pull request merged**